clients/dataflow/lib/google_api/dataflow/v1b3/model/big_table_io_details.ex

```diff
@@ -12,41 +12,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# NOTE: This class is auto generated by the swagger code generator program.
-# https://github.com/swagger-api/swagger-codegen.git
+# NOTE: This class is auto generated by the elixir code generator program.
 # Do not edit the class manually.
 
-defmodule GoogleApi.Dataflow.V1b3.Model.BigTableIoDetails do
+defmodule GoogleApi.Dataflow.V1b3.Model.BigTableIODetails do
```

Comment:
@chingor13 I noticed this (`Io`/`IO`). Is this something we're doing deliberately?

Reply:
Looks like the name change comes from the discovery doc.